src/pages/Login/components/LoginBtn.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function LoginBtn() {
+  const ApiKey = process.env.REACT_APP_API_KEY;
+  const RedirectUri = process.env.REACT_APP_REDIRECT_URI;
+  const KAKAO_AUTH_URL = `https://kauth.kakao.com/oauth/authorize?client_id=${ApiKey}&redirect_uri=${RedirectUri}&response_type=code`;
```

Comment:
```suggestion
  const KAKAO_AUTH_URL = `https://kauth.kakao.com/oauth/authorize?client_id=${process.env.REACT_APP_API_KEY}&redirect_uri=${process.env.REACT_APP_REDIRECT_URI}&response_type=code`;
```
해당 부분은 굳이 변수로 만들어서 작성하지 않아도 좋습니다~

Reply:
수정 완료